templates/backup.service.j2

```diff
@@ -0,0 +1,60 @@
+# Managed by Ansible, please don't edit manually
+
+[Unit]
+Description=borgmatic backup
```

Comment:
How about calling this `borgmatic.service`? More specific than `backup.service`. There could be other backup scripts.

Reply:
o.k. done